netbox/users/graphql/types.py

```diff
@@ -1,38 +1,45 @@
-from django.contrib.auth import get_user_model
-from graphene_django import DjangoObjectType
+from typing import List
 
+import strawberry
+import strawberry_django
+from django.contrib.auth import get_user_model
+from django.contrib.auth.models import Group
+from strawberry import auto
 from users import filtersets
 from users.models import Group
 from utilities.querysets import RestrictedQuerySet
+from .filters import *
 
 __all__ = (
     'GroupType',
     'UserType',
 )
 
 
-class GroupType(DjangoObjectType):
-
-    class Meta:
-        model = Group
-        fields = ('id', 'name')
-        filterset_class = filtersets.GroupFilterSet
-
+@strawberry_django.type(
+    Group,
+    fields=['id', 'name'],
+    filters=GroupFilter
+)
+class GroupType:
     @classmethod
-    def get_queryset(cls, queryset, info):
-        return RestrictedQuerySet(model=Group).restrict(info.context.user, 'view')
+    def get_queryset(cls, queryset, info, **kwargs):
+        return RestrictedQuerySet(model=Group).restrict(info.context.request.user, 'view')
```

Comment:
Is a custom `get_queryset()` method still needed for the User & Group models now that we have our own custom models in place?